source/fundamentals/network-compression.txt

```diff
@@ -0,0 +1,97 @@
+.. _golang-network-compression:
+
+===================
+Network Compression
+===================
+
+In this guide, you can learn how to enable **network compression** by using
+the {+driver-short+}. You can specify a driver option to compress messages,
+which reduces the amount of data passed over the network between MongoDB and
+your application.
+
+The {+driver-short+} supports the following compression algorithms:
+
+1. `Snappy <https://google.github.io/snappy/>`__: available in MongoDB 3.4 and later.
+
+2. `Zlib <https://zlib.net/>`__: available in MongoDB 3.6 and later.
+
+3. `Zstandard <https://github.com/facebook/zstd/>`__: available in MongoDB 4.2 and later.
+
+If you specify multiple compression algorithms, the driver selects the
+first one in the list supported by your MongoDB instance.
+
+.. note::
+
+   When using the Snappy or Zstandard compression algorithm, you must
+   :ref:`add explicit dependencies <compression-dependencies>`.
+
+.. _enable-compression:
+
+Specify Compression Algorithms
+------------------------------
+
+You can enable compression for the connection to your MongoDB instance
+by specifying the algorithms in one of two ways: 
```

Comment:
Suggestion:
I think it could be good to use something similar to the [All List Items Are Sentences](https://www.mongodb.com/docs/meta/style-guide/style/lists/#all-list-items-are-sentences) style guide example to introduce the algorithms:
> You can do only one of the following:
>   Create multiple private endpoints in a single region.
>   Create one private endpoint in multiple regions.

E.g.
You can enable compression for the connection in one of the following ways:
- Add ...
- Specify ...